includes/discovery/bgp-peers.inc.php

```diff
@@ -5,6 +5,71 @@
 use LibreNMS\Util\IP;
 
 if (Config::get('enable_bgp')) {
+    if ($device['os'] == 'timos') {
+        $bgpPeersCache =snmpwalk_cache_multi_oid($device, 'tBgpPeerNgTable', [], 'TIMETRA-BGP-MIB', 'nokia');
```

Comment:
the mib directory 'nokia' is extraneous here, it is already defined in the os yaml.

Reply:
in the timos.yaml is defined group: nokia, I think that what you mean